config.w32

```diff
@@ -281,6 +286,10 @@ if (PHP_MONGODB != "no") {
     );
   } else if (PHP_MONGODB_CLIENT_SIDE_ENCRYPTION != "no") {
     WARNING("mongodb libmongocrypt support not enabled, crypto libs not found");
+
+    // Add kms_message sources bundled with libmongoc
```

Comment:
In this case `mongoc_ssl_found` is `false`. Does it make sense to compile KMS here?

Previously, the KMS defines were only set on CFLAGS when `mongoc_ssl_found` was `true`.

If I had to guess, you'd likely only want to use libmongoc's KMS sources when libmongocrypt is disabled but `mongoc_ssl_found` is `true`.

Reply:
That is correct, and config.m4 was affected as well. Fixing this in #1108.